src/widgets/Typography/TextWithIcon.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/forbid-prop-types */
+/* eslint-disable arrow-body-style */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text } from 'react-native';
+
+import { FlexRow } from '../FlexRow';
+
+import { APP_FONT_FAMILY, DARKER_GREY } from '../../globalStyles';
+
+const SIZES = {
+  xs: 10,
+  s: 12,
+  ms: 13,
+  m: 14,
+  ml: 16,
+  l: 20,
+};
+
+const getInternalStyle = (size, left, color, textStyle) => {
+  const margin = left ? 'marginLeft' : 'marginRight';
+  const alignment = left ? 'left' : 'right';
+
+  const styleAdjustment = {
+    fontSize: SIZES[size],
+    [margin]: 5,
```

Comment:
is the margin value ok at larger sizes?

Reply:
I originally had the margin equal to the font size, but having the hard 5 seemed to look better.. 🤷